client/src/components/atoms/Button.tsx

```diff
@@ -3,15 +3,17 @@ import React, { FunctionComponent } from "react";
 import {CopyToClipboard} from 'react-copy-to-clipboard';
 
 interface ButtonProps {
+    className?: string,
     text: string,
     onClick?: React.MouseEventHandler<HTMLButtonElement>,
-    copyText: string
+    copyText: string,
+    type?: "button" | "submit" | "reset" | undefined,
```

Comment:
Just curious, are these going to be the only types of buttons we'll have?

Reply:
Yup! they're taken from the type attribute on html buttons (see [here](https://www.w3schools.com/tags/att_button_type.asp))